### ISSUE TITLE
Crashing after log in on Monterey 12.6.3

### ISSUE THREAD

**Issue by codeanpeace** [lbrndnr/nuage-macos#27]
Hello, nuage is repeatedly crashing beginning after the very first log in following an install via homebrew.

Stacktrace from crash report:

> Crashed Thread:        0  Dispatch queue: com.apple.main-thread
> 
> Exception Type:        EXC_BREAKPOINT (SIGTRAP)
> Exception Codes:       0x0000000000000001, 0x0000000100252778
> Exception Note:        EXC_CORPSE_NOTIFY
> 
> Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
> Terminating Process:   exc handler [5714]
> 
> Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
> 0   Nuage                         	       0x100252778 closure #3 in MainView.body.getter + 2012
> 1   Nuage                         	       0x1002524e0 closure #3 in MainView.body.getter + 1348
> 2   SwiftUI                       	       0x1b21b80f8 closure #1 in _AppearanceActionModifier.MergedBox.update() + 100
> 3   SwiftUI                       	       0x1b1a138f0 thunk for @escaping @callee_guaranteed () -> () + 28
> 4   SwiftUI                       	       0x1b230924c closure #1 in ViewRendererHost.render(interval:updateDisplayList:) + 2532
> 5   SwiftUI                       	       0x1b22f6b28 ViewRendererHost.render(interval:updateDisplayList:) + 388
> 6   SwiftUI                       	       0x1b2359214 closure #1 in NSHostingView.layout() + 212
> ...

**Comment by lbrndnr** [comment]
Thanks for your feedback! I think i have already fixed this issue in the latest release. Would you mind compiling the master branch to verify?

**Comment by lbrndnr** [comment]
I have just released a new version that fixes a crash upon login. Could you quickly verify that 0.0.5 fixes this? :)

**Comment by codeanpeace** [comment]
Yup, I can confirm that the new release no longer crashes on launch.

Some observations:
- stream tab is empty
- searching via cmd-l keyboard doesn't focus the search bar
- filtering via cmd-f works but randomly caused a crash, haven't yet reproduced though
- playlists dropdown mixes together user created playlists with liked playlists created by other users

Regardless, thanks a bunch for working on a modern Mac app for Soundcloud!

**Comment by lbrndnr** [comment]
stream tab is empty, can you elaborate? does it remain empty or is just empty on first launch? And could you explain the last point as well?

**Comment by codeanpeace** [comment]
<img width="811" alt="image" src="https://github.com/lbrndnr/nuage-macos/assets/2991389/ed3d5128-aed2-4c2d-8ffd-8b6db4acd8d2">

**Comment by codeanpeace** [comment]
Regarding the playlists, there are two types of playlists on SoundCloud – those that you created yourself and those that you liked from others. If you visit [https://soundcloud.com/you/sets](https://soundcloud.com/you/sets), you'll see a dropdown on the right that defaults to "All" but also lets offers "Liked" and "Created" as options. It'd be nice if nuage could build in that distinction as I'd much rather only see those that I created myself. While I've only created a dozen or so playlists, I've liked dozens of other artists' and users' playlists/albums/sets. Hope this helps!
